package.json

```diff
@@ -105,7 +105,8 @@
   },
   "resolutions": {
     "@types/react": "17.0.66",
-    "@graasp/sdk": "2.0.0"
+    "@graasp/sdk": "https://github.com/graasp/graasp-sdk.git#library-triggers-enum",
+    "immutable": "4.3.4"
```

Comment:
Todo: rebase on main, @LinaYahya if you need help with that ping us, as we made quite a lot of changes

Reply:
Done
@spaenleh